### ISSUE TITLE
Size reporting seems broken

### ISSUE THREAD

**Issue by benlesh** [ReactiveX/rxjs#2907]
The sizes reported by DangerJS seem off. It should be comparing the size of the output files from the PR against the size of the output files from master. I have a feeling this isn't the case.

Also it's worth investigating other approaches.

cc @kwonoj

**Comment by kwonoj** [comment]
@benlesh nah, at the moment we were agreed to use `latest release` vs. PR (https://github.com/ReactiveX/rxjs/blob/0939f61eaf131090724cd18012cea1891e311a52/dangerfile.js#L110-L114) so it's working as expected, but not the way you currently expected. :p 

compare to master is probably doable but need some more wiring, cause on travis I recall there's something with git history, also we need to build twice (master , PR) to get comparison result.

**Comment by kwonoj** [comment]
Let me labelling off `bug` but labelling `enhancement` .

**Comment by benlesh** [comment]
That sounds familiar... Okay. Thanks @kwonoj!
